lib/shipit/stack_commands.rb

```diff
@@ -71,6 +77,18 @@ def with_temporary_working_directory(commit: nil)
       end
     end
 
+    def valid_git_repository?(path)
+      Dir.exist?(path) &&
```

Comment:
I was about to say `path` is a `Pathname` so this code could be cleaned up, but turns out I'm wrong, for some reason `Stack#path` returns a `Pathname`, but `Stack#git_path` returns a `String`... I should clean that up.